### PR TITLE
Add new New York file

### PR DESCRIPTION
**Pull request by kwill307**
Adding base file for restaurant to do list

@struther what do you think?

### CODE REVIEW

File: kwill307-newyork.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Where to Eat
+- Sponge Docks
+- Dunedin Brewery
```
Comment (struther) [inline kwill307-newyork.md:3]: This sounds good do they have any good dark beers?

File: kwill307-newyork.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## Where to Eat
+- Sponge Docks
```
Comment (struther) [inline kwill307-newyork.md:2]: @kwill307 do they have this place in ny?

**Review by struther — COMMENTED** [review]
@kwill307 
:chart_with_downwards_trend:

File: kwill307-newyork.md (hunk 1 above)
Comment (kwill307) [reply kwill307-newyork.md:3]: no, but Founder's and Odd side have some good ones :beers:

Comment (JasonMick) [reply kwill307-newyork.md:3]: [jmick4] What about Southern Tier... can't forget the Creme Brulee Stout. :beer:

Comment (kwill307) [reply kwill307-newyork.md:3]: They do make a good stout! :yum:

Comment (kwill307) [reply kwill307-newyork.md:3]: img

**Pull request merged**